assets/src/edit-story/elements/text/frame.js

```diff
@@ -100,7 +100,7 @@ function TextFrame({ element: { id, content, ...rest }, wrapperRef }) {
     const handleMouseUp = (evt) => {
       const timingDifference = window.performance.now() - clickTime;
 
-      if (timingDifference > 300) {
+      if (timingDifference > 150) {
```

Comment:
150ms is a VERY short click. It was increased to 300ms due to a reported bug that it was too small. I think our problem is more related to the "drag" than speed of click. The coordinates have moved too far and that could be a better indicator.

Reply:
Hmm, interesting.

When I was testing then the number was mostly under `10` actually, when clicking and holding down then around `100`.

However, might be just how it is for me locally. Will reopen the issue then to investigate more.

Reply:
I think it has a lot to do with the type of pointer: mouse vs touchpad vs touch.